File: 02-Basic-Maths-for-DSA/GFG/02-Floor and Ceil of Integer Division/Solution-1.py
import math 

def floor(a,b):
    # return a//b
    return math.floor(a/b)

def ceil(a,b):
    if(a%b==0):
        return a//b
    return (a//b)+1
    # return math.ceil(a/b)


if __name__ == "__main__":
    a = int(input("Enter a number: "))
    b = int(input("Enter another number: "))

    print(f"Floor is {floor(a,b)} and Ceil is {ceil(a,b)}")

File: 02-Basic-Maths-for-DSA/Fraz/02-Reverse a number/Solution-1.js
let num = 1234567;

let d,
  rev = 0;
while (num > 0) {
  d = num % 10;
  rev = rev * 10 + d;
  num = Math.floor(num / 10);
}
console.log(rev);


// T.C. : O(log n)